packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -86,94 +212,110 @@ def create_client(
     """
     channel_pool = GrpcChannelPool()
     discovery_client = DiscoveryClient(grpc_channel_pool=channel_pool)
-    built_in_import_modules: List[str] = []
-    custom_import_modules: List[str] = []
 
     if all:
-        measurement_service_class = get_all_registered_measurement_service_classes(discovery_client)
+        directory_out = _resolve_output_directory(directory_out)
```

Comment:
FYI, this is identical in all 3 cases, but it will not be needed for `--list`, so I think it's ok to continue to duplicate it.